networkml/featurizers/README.md

```diff
@@ -0,0 +1,78 @@
+# Features in NetworkML Models
+
+Machine learning models require inputs (or "features" in the language of machine learning practitioners.) NetworkML, a machine learning model that operates on network traffic, is no different. This readme describes the features that networkML can create and can therefore be included in NetworkML's models. All features can either be created for each host present, which therefore creates a numeric representation of all the traffic flowing into and out of a host, or for each session, which describes a particular data exchange for two hosts defined by a 5-tuple of source and destinatio IP's, source and destination ports, and protocol.
+
+A key at the bottom of this page explains the symbols associated with each feature set.
+
+## Protocol-Specific Features
+
+IPv4 (b)
+
+IPv6 (b)
+
+Well-known Ethernet protocols (b) [Note: Each flag is assigned an individual boolean vector. See list of ethernet protocols at the bottom of the page.]
+
+TCP ports (i/o) \(P\) (P/NP) (b)
+
+UDP ports (i/o) (P/NP) (b)
+
+TCP flags (i/o) (b) [Note: Each flag is assigned an individual boolean vector.]
+
+IP flags (i/o) (b)
+
+IP differentiated services (i/o) (b)
+
+Well-known IP protocols (b) [Note: Each protocol is assigned an individual boolean vector. See list of IP protocols at the bottom of the page.]
+
```

Comment:
New section  - non-IP fatures.

Reply:
Change made. I think I put the features in the correct category now, but a second look by @anarkiwi  or @cglewis wouldn't hurt.